src/main/java/org/elasticsearch/common/jna/Kernel32Library.java

```diff
@@ -37,17 +36,21 @@
 
     private static ESLogger logger = Loggers.getLogger(Kernel32Library.class);
 
-    private Kernel32 internal;
+    private boolean loaded = false;
```

Comment:
Can we remove this boolean? There is only one constructor for this class, so if it succeeds, then we loaded. otherwise we have no object to call addConsoleCtrlHandler() on, so we can remove the check for it there, too.